test.sh

```diff
@@ -4,7 +4,7 @@ set -u # crash on missing env
 set -e # stop on any error
 
 echo "Running style checks"
-flake8
+flake8 --exclude=venv
 
```

Comment:
Just a question: why is this not needed in other projects?

Reply:
Because in other projects we have a `src`-dir, in which the `test.sh` resides, but not `venv`